cpp/src/io/parquet/decode_preprocess.cu

```diff
@@ -353,20 +353,17 @@ CUDF_KERNEL void __launch_bounds__(preprocess_block_size)
 
   // initialize the stream decoders (requires values computed in setupLocalPageInfo)
   // the size of the rolling batch buffer
-  int const max_batch_size = rolling_buf_size;
-  level_t* rep             = reinterpret_cast<level_t*>(pp->lvl_decode_buf[level_type::REPETITION]);
-  level_t* def             = reinterpret_cast<level_t*>(pp->lvl_decode_buf[level_type::DEFINITION]);
+  level_t* rep = reinterpret_cast<level_t*>(pp->lvl_decode_buf[level_type::REPETITION]);
+  level_t* def = reinterpret_cast<level_t*>(pp->lvl_decode_buf[level_type::DEFINITION]);
```

Comment:
```suggestion
  level_t *const rep = reinterpret_cast<level_t *const>(pp->lvl_decode_buf[level_type::REPETITION]);
  level_t *const def = reinterpret_cast<level_t *const>(pp->lvl_decode_buf[level_type::DEFINITION]);
```

Reply:
I can change the type, but the `reinterpret_cast` complains that the access modifier is meaningless in the cast.

Reply:
Is this supposed to be `level_t const*`? pointer to const, rather than a const pointer?

Reply:
I don't think so. We are passing these to `init` and they are going to be used by the rle_stream stuff to write decoded values, so we need them to be mutable.